packages/react-aria-components/src/Table.tsx

```diff
@@ -846,6 +851,7 @@ function TableColumnHeader<T>({column}: {column: GridNode<T>}) {
     style = {...style, width: layoutState.getColumnWidth(column.key)};
   }
 
+  // TODO: Column doesn't have isInteractive nor data-interactive since data-allows-sorting exists and conveys the same info. Open to opinions
```

Comment:
open question, in the same vein as https://github.com/adobe/react-spectrum/pull/5559/files#r1425781145. I could see an argument for adding `data-interactive` for parity

Reply:
seems fine to me for now, we can always add later.